src/common/SPELLS/classic/engineering.ts

```diff
@@ -1,11 +1,41 @@
 import { spellIndexableList } from '../Spell';
 
 const spells = spellIndexableList({
+  COBALT_FRAG_BOMB: {
+    id: 67769,
+    name: 'Cobalt Frag Bomb',
+    icon: 'inv_misc_enggizmos_31',
+  },
+  FRAG_BELT: {
+    id: 67890,
+    name: 'Cobalt Frag Bomb',
```

Comment:
Should this also be `Cobalt Frag Bomb` if the key name is `FRAG_BELT`?

Reply:
It's a Cobalt Frag Bomb made from the Frag Belt. There's normal Cobalt Frag Bombs too.